plugins/sumneko_plugin.lua

```diff
@@ -7,23 +7,31 @@ local liquipedia = {}
 
 local importFunctions = {}
 importFunctions.functions = {'require', 'mw%.loadData', 'Lua%.import', 'Lua%.requireIfExists'}
-importFunctions.prefixModules = {table = 'standard.', math = 'standard.', string = 'standard.'}
+importFunctions.prefixModules = {table = 'standard.', math = 'standard.', string = 'standard.', array = 'standard.'}
```

Comment:
These refer to the standard, builtin libraries (e.g. `table.insert`), why do we add array here? Won't this break importing for `Module:Array`? (And `Module.Table` as well?)

Reply:
standard.table means `standard/table.lua`, so it would include ours, instead of the the lua standard package `table`

Reply:
So eventually every package in standard would be listed here, unless there is some automation to calculate these, right?

Reply:
Array was added as `luassert` has `array.lua` as well, so specify for the intellisense that we want our `standard/array.lua`, and not luassert's array.lua, by default

Reply:
> So eventually every package in standard would be listed here, unless there is some automation to calculate these, right?

Only where there is a necessity to specify, normally. Lua has a list of `paths` it will scan through, and it will find it

./src/?.lua
./src/?/?.lua
./src/?/init.lua
/home/rikard/.luarocks/share/lua/5.3/?.lua
/home/rikard/.luarocks/share/lua/5.3/?/init.lua
/usr/local/share/lua/5.3/?.lua
/usr/local/share/lua/5.3/?/init.lua
/usr/local/lib/lua/5.3/?.lua
/usr/local/lib/lua/5.3/?/init.lua
/usr/share/lua/5.3/?.lua
/usr/share/lua/5.3/?/init.lua
./?.lua
./?/init.lua

Reply:
I see, thank you for the explanation